apps/web/src/pages/workflow/index.tsx

```diff
@@ -146,32 +151,38 @@ const Workflow = () => {
         },
         [navigate],
     );
-    const handlerEditModal = (isAdd: boolean, isOpen: boolean, row?: EditFormDataProps): void => {
+    const handlerEditModal = (isAdd: boolean, isOpen: boolean, row?: EditRowData): void => {
```

Comment:
there is no need to pop up the edit modal here, just jump directly to the editor page